src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -83,8 +86,26 @@ public class HaplotypeCallerEngine implements AssemblyRegionEvaluator {
 
     protected final OutputStreamWriter assemblyDebugOutStream;
 
-    // the genotyping engine for the isActive() determination
-    private MinimalGenotypingEngine activeRegionEvaluationGenotyperEngine = null;
+    /**
+     * List of interval file entries including regions and custom ploidy values to apply in that region.
+     */
+    protected final List<NamedFeature> ploidyRegions = new ArrayList<>();
+
+    /**
+     * List of all custom ploidies provided by user
+     */
+    private final LinkedHashSet<String> allCustomPloidies;
```

Comment:
why is this a string?

Reply:
Switched to integers since `SimpleCount` exists and is great